tldw_Server_API/app/core/Watchlists/alert_rules.py

```
@@ -0,0 +1,244 @@
+            job_id=job_id,
+            name=name,
+            enabled=True,
+            condition_type=condition_type,
+            condition_value=cv,
+            severity=severity,
+            created_at=now,
+            updated_at=now,
+        )
+
+
+def update_alert_rule(db_path: str, rule_id: int, user_id: str, **fields: Any) -> bool:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**fields`

(ANN401)